src/hello-world/BUILD.bazel

```diff
@@ -14,11 +15,31 @@ sass_binary(
 
 ng_module(
     name = "hello-world",
-    srcs = glob(["*.ts"]),
+    srcs = glob([
```

Comment:
It seems a bit redundant to use a glob when you specify the files individually

Reply:
good catch - I don't have the skylark linter hooked up yet